src/main/java/pluto/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package pluto.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    protected static final String ANSI_RESET = "\u001B[0m";
+    protected static final String ANSI_RED = "\u001B[31m";
+    protected static final String ANSI_GREEN = "\u001B[32m";
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[%s] %s", this.getStatusIcon(), this.description);
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsUndone() {
+        this.isDone = false;
+    }
+
+
+    public abstract String toFile();
```

Comment:
I really like this method that you used to turn tasks into storage. If given the time, I would want to try to do the same as well. The idea is really solid.

My only concern is an ignorant or malicious user. What if they run a command like 
todo try | have to complete project